src/frontends/pytorch/src/op/batch_norm.cpp

```diff
@@ -63,7 +67,13 @@ OutputVector translate_batch_norm(const NodeContext& context) {
         bias = broadcast_const_to_channel_dim(context, input, zero_f);
     }
     // index 3 running_mean and index 4 running_var can be none for training case only, check that not training before
-    auto training = context.const_input<bool>(5);
+    bool training;
+    if(is_bn_no_training){
+        training = false;
+    }
+    else{
+        training = context.const_input<bool>(5);
```

Comment:
In my opinion better would be to extract training argument before calling `translate_batch_norm_common`

Reply:
Makes sense Maxim, then i can check for the num of inputs and get the value for epsilon. Made the changes